packages/itwinui-css/src/slider/slider.scss

```diff
@@ -9,6 +9,7 @@ $tick-height: var(--iui-size-s); // 12px
 .iui-slider-component-container {
   display: flex;
   align-items: flex-start;
+  flex-direction: unset;
```

Comment:
?

what does slider have to do with input containers?

Reply:
ugh, html slider had input-group. 
Fixed by removing iui-input-group in html.